www/audit.py

```diff
@@ -728,6 +745,7 @@ def api_feature(pid):
                 elif not user_did_it:
                     feat.validates_count += 1
                 feat.save()
+                update_csv(csv_file, project, user, ref_and_audit[0], ref_and_audit[1])
```

Comment:
Die Elemente von `ref_and_audit` können hier direkt in 2 Argumente entpackt werden:
```suggestion
                update_csv(csv_file, project, user, *ref_and_audit)
```

(Noch sinnvoller wäre aber wahrscheinlich, schon weiter oben die Rückgabe von `request.get_json()` in zwei Variablen zu destrukturieren. Falls es immer zwei Elemente drin haben muss, und das erste `ref` und das zweite `audit` ist (ist dem so?), dann geht das mit:
```
ref, audit = request.get_json()
```